src/packagedcode/pypi.py

```diff
@@ -270,16 +270,22 @@ def parse_setup_py(location):
     parties = []
     author = setup_args.get('author')
     author_email = setup_args.get('author_email')
+    homepage_url = setup_args.get('url')
     if author:
         parties.append(
             models.Party(
                 type=models.party_person,
                 name=author,
                 email=author_email,
-                role='author'
+                role='author',
+                url=homepage_url
```

Comment:
Is this URL really the URL for an author? Or instead that for the package?

Reply:
It is the package url